src/Abp.OnlineClientStore.Redis/RealTime/Redis/AbpRedisOnlineClientStoreOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Abp.Configuration.Startup;
+
+namespace Abp.RealTime.Redis
+{
+    public class AbpRedisOnlineClientStoreOptions : IAbpRedisOnlineClientStoreOptions
```

Comment:
How is this different from `AbpRedisCacheOptions` ?

Reply:
It's not

Reply:
In that case, can we reuse `AbpRedisCacheOptions`?

Reply:
Yes if we move redis online client store to RedisCache project 🙂